.github/workflows/tests.yml

```diff
@@ -0,0 +1,56 @@
+name: "CI"
+on: [push, pull_request]
+
+jobs:
+  bats:
+    name: bats
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Setup bats
+        uses: mig4/setup-bats@v1
+        with:
+            bats-version: 1.9.0
+
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Configure git (user name)
+        run: git config --global user.name "Decomposer (CI)"
+
+      - name: Configure git (user email)
+        run: git config --global user.email "actions@php-decomposer.org"
+
+      - name: Configure git (default branch)
+        run: git config --global init.defaultBranch master
+
+      - name: Run bats
+        run: bats --print-output-on-failure -r tests
+
+  jsonschema:
+    name: jsonschema
+    runs-on: ubuntu-latest
```

Comment:
I think this might be better in a separate workflow. I think the curl is not enough and we need an actual git clone to work with, and that would open up the road to a testsuite where we clone and test with multiple different repos.